### PR TITLE
Adds support for the COMPREHENSIVE_THEME_LOCALE_PATHS setting

### PR DESCRIPTION
**Pull request by douglashall**
## Configuration Pull Request

Adds support for setting the `COMPREHENSIVE _THEME_LOCALE_PATHS` setting added with https://github.com/edx/edx-platform/pull/13492.

Make sure that the following steps are done before merging
- [x] @devops team member has commented with :+1:
- [x] are you adding any new default values that need to be overriden when this goes live?  
  - [x] Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
  - [x] Add an entry to the CHANGELOG.

### CODE REVIEW

**Comment by douglashall** [comment]
FYI @fredsmith

**Comment by douglashall** [comment]
https://openedx.atlassian.net/browse/DEVOPS-4918 created to add configuration_secure overrides.

File: util/jenkins/ansible-provision.sh [hunk 1]
```diff
@@ -185,6 +185,7 @@ edx_platform_repo: ${edx_platform_repo}
 
 EDXAPP_PLATFORM_NAME: $sandbox_platform_name
 EDXAPP_COMPREHENSIVE_THEME_DIRS: $edxapp_comprehensive_theme_dirs
+EDXAPP_COMPREHENSIVE_THEME_LOCALE_PATHS: $edxapp_comprehensive_theme_locale_paths
```
Comment (jibsheet) [inline util/jenkins/ansible-provision.sh]: This assumes that the ansible provision Jenkins job will be updated to have a new dropdown selector or text input for this.  Is that true.  If so, you will need to file a ticket about this and the DSL being used on the new jenkins.edx.org will require updating.

Comment (douglashall) [reply util/jenkins/ansible-provision.sh]: Got it. Thanks for explaining the purpose of this config. I have removed it.

**Review by jibsheet — APPROVED** [review]

**Pull request merged**